src/Compilers/CSharp/Test/Syntax/Parsing/NullableParsingTests.cs

```diff
@@ -1181,11 +1232,265 @@ public void NullableArray_PartialMember()
                     N(SyntaxKind.MethodDeclaration);
                     {
                         N(SyntaxKind.PartialKeyword);
+                        N(SyntaxKind.NullableType);
+                        {
+                            N(SyntaxKind.ArrayType);
+                            {
+                                N(SyntaxKind.IdentifierName);
+                                {
+                                    N(SyntaxKind.IdentifierToken, "A");
+                                }
+                                N(SyntaxKind.ArrayRankSpecifier);
+                                {
+                                    N(SyntaxKind.OpenBracketToken);
+                                    N(SyntaxKind.OmittedArraySizeExpression);
+                                    {
+                                        N(SyntaxKind.OmittedArraySizeExpressionToken);
+                                    }
+                                    N(SyntaxKind.CloseBracketToken);
+                                }
+                            }
+                            N(SyntaxKind.QuestionToken);
+                        }
+                        N(SyntaxKind.IdentifierToken, "F");
+                        N(SyntaxKind.ParameterList);
+                        {
+                            N(SyntaxKind.OpenParenToken);
+                            N(SyntaxKind.CloseParenToken);
+                        }
+                        N(SyntaxKind.SemicolonToken);
+                    }
+                    N(SyntaxKind.CloseBraceToken);
+                }
+                N(SyntaxKind.EndOfFileToken);
+            }
+            EOF();
+        }
+
+        [Fact]
+        public void CreateNullableArray_01()
+        {
+            UsingExpression("new object[,][]?",
```

Comment:
I'm surprised this is a parse error. I would have thought it parsed fine, but produced a semantic error